cmd/installer/cli/install2.go

```diff
@@ -211,6 +211,8 @@ func preRunInstall2(cmd *cobra.Command, flags *Install2CmdFlags) error {
 			return fmt.Errorf("unable to parse license file: %w", err)
 		}
 		flags.license = l
+	} else if requireLicense {
```

Comment:
Can you add this as well to addInstallFlags and MarkFlagRequired?

Reply:
There was a 'uncomment when we get tests passing' for requiring the license flag
I've done that now instead of this 'requireLicense' stuff